dockerfiles/README.md

```diff
@@ -0,0 +1,10 @@
+# Obscuro enclave service Docker image
+
+To create the Docker image for an Obscuro enclave service running in SGX simulation mode:
+
+    docker build -t obscuro_enclave - < dockerfiles/enclave.Dockerfile
+
+To run the image as a container, where `XXXX` is the port on which to expose the enclave service's RPC endpoints on the 
+local machine, and `YYY` is the address of the node that this enclave service is for as an integer:
+
+    docker run -p XXX:11000/tcp obscuro_enclave --nodeID YYY
```

Comment:
are there any instructions for running the test?

Reply:
This is about building the enclave Docker image. The test is self-contained, you just run it as normal.

Reply:
basically all you need to do to run the test is to have "docker" running?

Reply:
Docker running, and the image published.

Reply:
that requires instructions. Probably common sense stuff if you're a docker user, but still

Reply:
Ok, I've added a line, I hope that's what you had in mind.